components/raftstore/src/store/config.rs

```diff
@@ -586,6 +609,9 @@ impl Default for Config {
             enable_v2_compatible_learner: false,
             unsafe_disable_check_quorum: false,
             min_pending_apply_region_count: 10,
+            check_then_compact_force_bottommost_level: true,
+            check_then_compact_top_n: 10,
```

Comment:
Can we adjust dynamically? I'm afraid it would not be adequate when regions are a lot.